.gitignore

```diff
@@ -1,7 +1,8 @@
-hnswlib.egg-info/

```

Comment:
I think the file was originally created on Windows? It had `\r` in the file. My only intended change was to add the `var/` ignore. I can remove this change if desired.

Reply:
@groodt Yes, it was created on Windows.